repos/fdbt-netex-output/serverless.yml

```diff
@@ -65,6 +65,11 @@ provider:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
+      lifecycleConfiguration:
+        Rules:
+          - Id: ExpiryRule
+            Status: Enabled
+            ExpirationInDays: '90'
```

Comment:
I think we'll need this on the matching-data bucket?

Reply:
hmm, I thought we needed it for the bucket where the exports sit?

Reply:
The "to be" exported files sit in the matching-data bucket and the completed exported files sit in the netex-data bucket. So the list of exports is driven from the matching-data bucket. No harm in also clearing up these ones here though I don't think